tools/tertiary-analysis/scanpy/scanpy-neighbours.xml

```diff
@@ -51,7 +51,7 @@ PYTHONIOENCODING=utf-8 scanpy-neighbors
       <when value="true"/>
       <when value="false">
         <param name="key_added" argument="--key-added" type="text" optional="true" label="Key added"
-            help="If not specified, the neighbors data is stored in .uns[‘neighbors’], distances and connectivities are stored in .obsp[‘distances’] and .obsp[‘connectivities’] respectively. If specified, the neighbors data is added to .uns[key_added], distances are stored in .obsp[key_added+’_distances’] and connectivities in .obsp[key_added+’_connectivities’]." value='neighbours' />
+            help="If not specified, the neighbors data is stored in .uns[‘neighbors’], distances and connectivities are stored in .obsp[‘distances’] and .obsp[‘connectivities’] respectively. If specified, the neighbors data is added to .uns[key_added], distances are stored in .obsp[key_added+’_distances’] and connectivities in .obsp[key_added+’_connectivities’]." value='' />
```

Comment:
we are removing the default but scanpy internally will keep setting the default to neighbours, correct?

Reply:
maybe we should add some changes later on scanpy scripts to store metadata of umap, tsne and other in uns as it is done with the clusters here.

Reply:
Annoyingly the metadata is not actually stored sensibly even for clusters (the slot used seems fixed), so we'd need to do some work with Scanpy itself. I think we should do that, but maybe we need something quicker than that would be possible.

Reply:
... and yes, scanpy will default to 'neighbors'. Unfortunately by specifying it you trigger some different behaviour than is assumed by downstream steps when the default value is in place. Another bug to address in Scanpy.